operators/pkg/exmagnt/template.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020-2022 Politecnico di Torino
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exmagnt
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	clv1alpha2 "github.com/netgroup-polito/CrownLabs/operators/api/v1alpha2"
+)
+
+// EATemplateHandler is the handler for the EATemplate.
+type EATemplateHandler struct{}
+
+const (
+	// EATemplateEndpoint is the endpoint for the EAInstance.
+	EATemplateEndpoint = "/template"
+)
+
+// EATemplate represents a Template within the exmagnt.
+type EATemplate struct {
+	Name       string `json:"name"`
+	PrettyName string `json:"prettyName"`
+	CreatedAt  string `json:"createdAt"`
```

Comment:
You can probably leave this as timestamp, and then it is formatted automatically.